src/main/resources/bank/items/.gitignore

```diff
@@ -0,0 +1,3 @@
+# ignore everything except .gitignore
```

Comment:
I don't think this is needed.  I believe you can just add `src/main/resources/bank/**` to the main .gitignore file.  That'll ignore all the files in /bank which is what I think you want.